src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+  let smallDiscount = 20;
+  let bigDiscount = 50;
```

Comment:
Consider using descriptive constant names for the values used in the calculation. This will make the code more readable and maintainable. For example, use constants like DAILY_RATE, LONG_TERM_DAYS, LONG_TERM_DISCOUNT, SHORT_TERM_DAYS, and SHORT_TERM_DISCOUNT.